myning/chapters/__init__.py

```diff
@@ -30,7 +30,18 @@ class ExitArgs:
 PickHandler = Callable[..., PickArgs]
 Handler = Callable[..., PickArgs | DynamicArgs | ExitArgs]
 OptionLabel = str | Text | list[str | Text]
-Option = tuple[OptionLabel, Handler]
+
+
+@dataclass
+class Option:
+    label: OptionLabel
+    handler: Handler
+    enable_hotkeys: bool = False
```

Comment:
I think it would be better to have this default to True, because we generally want to have hotkeys, and not enabling them is the exception. Thinking about if we add more menus in the future, we can remove the cognitive load of having to enable hotkeys, and just turn them off when necessary

The way to make this change would be to remove all the places we've set it to True (could probably search and replace) and then just adding false to the few menus

This will also make it easier in the future to search for which menus have hotkeys disabled (they're usually created in a list comprehension e.g. list of army members)

Also I prefer implicit typing when possible, but removing the type hint is not a blocker

Reply:
I agree on `True` for the default. I like the explicit typing tho

Reply:
I am good with default True. It seems about the same amount of work having it default to True or False to me, so I just took the standard bool default. I am with Caleb on explicit on function and class definitions.

Reply:
I am happy with explicit typing on a dataclass property